<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infdeduction</class>
 <widget class="QMainWindow" name="infdeduction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1" rowspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>机载红外图像：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fileline"/>
          </item>
          <item>
           <widget class="QPushButton" name="browser">
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>波段：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="phase">
            <item>
             <property name="text">
              <string>近红外</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>中红外</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>远红外</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>机载图像分辨率（m）：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="res">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>星载图像分辨率（m）：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="spe_res">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>保存路径：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="save_path_line"/>
          </item>
          <item>
           <widget class="QPushButton" name="browser_2">
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>保存名称：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="save_name"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>注意：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>机载红外图像需要提前放入项目文件夹中</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>运行日志：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="log"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="run">
            <property name="text">
             <string>运行</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="save">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="initialize">
            <property name="text">
             <string>初始化</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exit">
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>机载红外图像（预览）</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="src_img">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>星载红外图像（预览）</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="out_img">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>37</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>红外图像推演模块</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1197</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
